projects/core/src/cart/facade/selective-cart.service.ts

```diff
@@ -36,12 +38,21 @@ export class SelectiveCartService {
     protected store: Store<StateWithMultiCart>,
     protected userService: UserService,
     protected authService: AuthService,
-    protected multiCartService: MultiCartService
+    protected multiCartService: MultiCartService,
+    protected baseSiteService: BaseSiteService
   ) {
+    if (this.baseSiteService) {
```

Comment:
I would drop the `if` as it will be released in the same version the service was published in the public API.